src/test/scripts/_unittest_verbose.bat

```diff
@@ -0,0 +1,9 @@
+@ECHO OFF
+
+IF "%~1"=="" (
+"%~dp0XbSymbolDatabaseUnitTest.exe" --verbose
+) ELSE (
+"%~dp0XbSymbolDatabaseUnitTest.exe" "%~1" --verbose
+)
+
+PAUSE
```

Comment:
Why `bat` if there's also bash for this?
Why not powershell (which should integrate `bash` if the user enabled it)?

Reply:
bat == Windows platform
bash == Linux/other platforms

I had not heard about powershell has bash integrate. Sounds like WSL requirement which I don't use.

I mainly use drag 'n drop to check test results quicker than manually type in. The same for cache script as well.

Reply:
Ah, yes. Looks like it requires WSL. Oops.